optimalAssignment.py

```diff
@@ -1,4 +1,40 @@
+from itertools import combinations
+from itertools import permutaions
+
+"""
+botL --> list of available botID's
+taskD --> Dictionary with keys as taks names and values as the position to go ( as an object of the class Vector2D )
+"""
+
 class optimalAssignment(object):
-	def __init__(self, arg):
-		pass
-		
\ No newline at end of file
+	def __init__(self, state, botL, taskD):
+		self.Mat = self.assign( state, botL, taskD)
```

Comment:
remove extra space between `(` and `state`.

Reply:
yup